### PR TITLE
Readme updated with traceability changes

### PR DESCRIPTION
**Pull request by Josh-01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,25 +385,20 @@ jobs:
...
       with:
         login-server: contoso.azurecr.io
-        username: ${{ env.CR_USERNAME }}
-        password: ${{ env.CR_PASSWORD }}
+        username: ${{ secrets.REGISTRY_USERNAME }}
+        password: ${{ secrets.REGISTRY_PASSWORD }}
     
     - run: |
-        docker build . -t contoso.azurecr.io/k8sdemo:${{ github.sha }} --label dockerfile-path=./Dockerfile
+        docker build . -t contoso.azurecr.io/k8sdemo:${{ github.sha }} --label dockerfile-path=https://github.com/Username/RepoName/blob/branchOrTag/Dockerfile
```
Comment (ds-ms) [inline README.md:392]: dockerfile-path=https://github.com/${{github.repo}}/blob/${GITHUB_REF##*/}/Dockerfile

**Review by ds-ms — APPROVED** [review]
Consider updating the sample yaml here: https://github.com/Azure/actions-workflow-samples/blob/master/Kubernetes/build-and-deploy-docker-image-to-aks-using-helm.yml

File: README.md (hunk 1 above)
Comment (Josh-01) [reply README.md:392]: What does `${GITHUB_REF##*/}` indicate ?

Comment (ds-ms) [reply README.md:392]: GITHUB_REF is an environment variable right. It extracts out the branch name like this:

image

Comment (Josh-01) [reply README.md:392]: Thanks for the suggestion, I will change that.

**Pull request merged**